utility/src/main/java/org/oppia/android/util/networking/ProdNetworkConnectionUtil.kt

```diff
@@ -0,0 +1,36 @@
+package org.oppia.android.util.networking
+
+import android.content.Context
+import android.net.ConnectivityManager
+import org.oppia.android.util.networking.NetworkConnectionUtil.ConnectionStatus
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * [NetworkConnectionUtil] which gets the current [ConnectionStatus] of the device in production builds.
+ */
+@Singleton
```

Comment:
This isn't needed since there isn't any state being stored in the class.

Reply:
Removed.